src/main/java/duke/parser/Parser.java

```diff
@@ -64,13 +62,11 @@ public String parse(String input) {
             if (input.toLowerCase().equalsIgnoreCase("list")) {

```

Comment:
can consider using switch/case statements instead of if/else? This will make the code look more readable